scripts/protect_branch_git_hook.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+import sys
+
+
+def ProtectMasterBranch():
+    hookid = "protect-master-branch"
+    protected_branch = b"master"
+    current_branch = (
+        subprocess.run(["git", "symbolic-ref", "HEAD"], capture_output=True)
+        .stdout.split(b"/")[-1]
```

Comment:
We currently don't have protected branches containing `/`, but that's a possibility in future. Perhaps using `git branch --show-current` would eliminate the need for parsing output.

Reply:
Works like a charm!